code/game/objects/items/stacks/tiles/tile_reskinning.dm

```diff
@@ -0,0 +1,54 @@
+/////////////////////
+// Tile reskinning //
+/////////////////////
+// Q: What is this?
+// A: A simple function to allow you to change what tiles you place with a stack of tiles.
+// Q: Why do it this way?
+// A: This allows players more freedom to do beautiful-looking builds. Having five types of titanium tile would be clunky as heck.
+// Q: Great! Can I use this for all floors?
+// A: Unfortunately, this does not work on subtypes of plasteel and instead we must change the icon_state of these turfs instead, as the icon_regular_floor var that "saves" what type of floor a plasteel subtype turf was so once repaired...
+// ... it'll go back to the floor it was instead of grey (medical floors turn white even after crowbaring the tile and putting it back). This stops changing turf_type from working.
+
+/obj/item/stack/tile/mineral/titanium/ShiftClick(mob/user)
+	var/choice = input(user, "What tile would you like?") as null|anything in list("Titanium", "Yellow Titanium", "Blue Titanium", "White Titanium", "Purple Titanium", "Titanium Tile", "Yellow Titanium Tile", "Blue Titanium Tile", "White Titanium Tile", "Purple Titanium Tile")
```

Comment:
Are you kind enough to make this a radial so I don't have to? 🙂

Reply:
Sure! I haven't touched anything regarding radial menus; the thought of radial menus didn't ever cross my mind but I'll look into other implementations of radial menus and I'll try and learn how to make it work for this.
I suppose DNM for now, it's learning time.

Reply:
The good news for a usecase like this is that it's a static radial, no need to come up with a dynamic radial option. Look at things like the I did with TCG cards for an example:
https://github.com/tgstation/tgstation/blob/2844c95561c81bff1b7cf2fc10d0503d32824112/code/game/objects/items/tcg/tcg.dm#L52

Reply:
![image](https://user-images.githubusercontent.com/53862927/95046030-2eefbd00-06db-11eb-9ca5-bfe899fe099d.png)
@Jared-Fogle 
Okay so it's working, but I think it can be improved further. Is there a way to set the amount of options per page of the radial menu?
There are two tiles, each tiles has 5 variants, if I could set it to only have 5 options per page, I can have one tile and it's variants only on each page, instead of cramming 7 options on one page and 3 on the other.

Reply:
> Is there a way to set the amount of options per page of the radial menu?

Increase the radius parameter when calling show_radial_menu.

Reply:
Radius does urh, I guess I should've expected radius increases the radius between the centre and buttons.
![image](https://user-images.githubusercontent.com/53862927/95047787-99562c80-06de-11eb-9945-94a9ad6449af.png)
It doesn't change how many buttons per page there are, heck.

Reply:
What if make 3 menu, first has 2 variants of tiles, and another 2 has 5 variants of color.

Reply:
@KathrinBailey Haven't done it myself, just code diving. I suppose if you can't figure it out, just leave it as is and in the future we can add a new parameter to show_radial_menu that lets you do this.